packages/nestjs-module/src/ogma.service.ts

```diff
@@ -23,24 +25,36 @@ export class OgmaService implements LoggerService {
   constructor(
     @InjectOgma() ogma?: Ogma,
     @Optional() @InjectOgmaContext() context?: string,
+    @Optional() private readonly requestContext?: RequestContext,
   ) {
     this.context = context || '';
     this.ogma = ogma ?? new Ogma();
   }
 
+  private getRequestId(
+    requestContext: RequestContext | Record<string, any>,
+  ): string | undefined {
+    if (typeof requestContext.getContext !== 'undefined') {
+      return requestContext.getContext().requestId;
+    }
+    return requestContext.requestId;
+  }
+
   /**
    * use Ogma to log at the INFO level
    * @param message What to print to the Ogma instance
    * @param context Optional context if you want to change what the original context was
+   * @param requestId Optional id of an request
    */
-  public info(message: any, context?: string): void {
-    this.printMessage(message, 'info', context);
+  public info(message: any, context?: string, requestId?: string): void {
+    this.printMessage(message, 'info', context, requestId);
   }
 
   /**
    * use Ogma to log at the ERROR level
    * @param message What to print to the Ogma instance
    * @param context Optional context if you want to change what the original context was
+   * @param requestId Optional id of an request
    */
   public error(message: any, context?: string): void {
     this.printMessage(message, 'error', context);
```

Comment:
There's no `requestId` on the `error` method. The docs decorator is there though, is this a miss?

Reply:
Yeah I forgot to mention. I found out that Nest.JS on error by default on third parameter inject name of error so in our scenario sets `requestId` which was unwanted so I decide to remove `requestId` from error cause we already have `printError` so is someone would like to print error with custom requestId could use that method instead. Do you have better idea to solve that issue? I couldn't find better